frontend/src/components/common/header/header.tsx

```diff
@@ -0,0 +1,153 @@
+import { FC, SearchRequestDto } from 'common/types/types';
+import { IconColor, IconName, AppRoute } from 'common/enums/enums';
+import { Link, Button, SimpleInput, Icon, Image } from '../common';
+import { useState, useAppForm } from 'hooks/hooks';
+import { getNameOf } from 'helpers/helpers';
+import { searchHeader as searchHeaderValidationSchema } from 'validation-schemas/validation-schemas';
+import { DEFAULT_SEARCH_PAYLOAD } from './common';
+import { DEFAULT_PICTURE } from 'common/constants/picture';
+
+import styles from './styles.module.scss';
+
+type Props = {
+  isActive: boolean;
+};
+
+const Header: FC<Props> = ({ isActive }) => {
+  const { control, handleSubmit } = useAppForm<SearchRequestDto>({
+    defaultValues: DEFAULT_SEARCH_PAYLOAD,
+    validationSchema: searchHeaderValidationSchema,
+  });
+
+  const handleSearchSubmit = (payload: SearchRequestDto): void => {
+    //
+    // eslint-disable-next-line no-console
+    console.log('payload', payload);
```

Comment:
Get rid of console.log()... Better to write a `TODO:..` comment to implement this function in the future...